oddt/toolkits/rdk.py

```diff
@@ -1091,6 +1092,60 @@ def __setstate__(self, state):
         self._res_dict = state['dicts']['res_dict']
 
 
+def diverse_conformers_generator(mol, n_conf=10, method='etkdg', seed=None):
+    """Produce diverse conformers using current conformer as starting point.
+     Each conformer is a copy of original molecule object.
+
+    Parameters
+    ----------
+    mol : oddt.toolkit.Molecule object
+        Molecule for which generating conformers
+
+    n_conf : int (default=10)
+        Targer number of conformers
+
+    method : string (default='etkdg')
+        Method for generating conformers. Supported methods: etkdg, etdg, dg.
+
+    seed : None or int (default=None)
+        Random seed
+
+    Returns
+    -------
+    mols : list of oddt.toolkit.Molecule objects
+        Molecules with diverse conformers
+    """
+    mol_clone = mol.clone
+    if method == 'etkdg':
```

Comment:
And here we can use kwargs

Reply:
I've exposed the RMSD filter to a `rmsd` argument. Which [other parameter](http://www.rdkit.org/Python_Docs/rdkit.Chem.rdDistGeom-module.html#EmbedMultipleConfs) would you like to expose?

Reply:
I think `maxAttempts` and `enforceChirality` might be useful.